tests/unit/type/list.tcl

```diff
@@ -387,12 +387,13 @@ start_server {
         assert {[r LPOS mylist c] == 2}
     }
 
-    test {LPOS RANK (positive and negative rank) option} {
+    test {LPOS RANK (positive, negative and zero rank) option} {
         assert {[r LPOS mylist c RANK 1] == 2}
         assert {[r LPOS mylist c RANK 2] == 6}
         assert {[r LPOS mylist c RANK 4] eq ""}
         assert {[r LPOS mylist c RANK -1] == 7}
         assert {[r LPOS mylist c RANK -2] == 6}
+        assert_error "*RANK can't be zero: use 1 to start from the first match, 2 from the second ... or use negative to start*" {r LPOS mylist c RANK 0}
```

Comment:
I feel that this error message is a bit too long, maybe we can just check for `can't be zero`?

Reply:
Hi Binbin, yes, I agree with you that the error message looks like a little bit longer. But I prefer to keep it from 2 reasons: 1. if possible, we should give client a more clearly and more specific string,  and 2. the previous error message includes how will happen if the rank is positve, so I add the message how it will happen if the rank is negative according to the document (https://redis.io/commands/lpos/) . Of course, we could check @oranagra or @yossigo suggestion which way is better. Thanks